src/lib/configs.ts

```diff
@@ -50,6 +50,26 @@ async function loadConfigsFromFile() {
   }
 }
 
+export function findSpecFile(baseDir): string{
+  const files = ['asyncapi.yaml', 'asyncapi.json', 'asyncapi.yml']
+  let indexOfFoundFile = -1
+
+  for (let index = 0; index < files.length; index++) {
+    const file = files[index]
+    if(existsSync(path.resolve(baseDir, file))){
+      if(indexOfFoundFile === -1){
+      indexOfFoundFile = index
+      } else {
+        throw new Error('Multiple AsyncAPI files found at the root of your project. Please make sure to choose one in your config file.')
+      }
+    }
+  }
+  
+  if(indexOfFoundFile === -1){
+    throw new Error('AsyncAPI file was not found at the root of your project. you can set its path in config file.')
+  }
+  return path.resolve(baseDir, files[indexOfFoundFile])
```

Comment:
This is a way to avoid the "Function Call Object Injection Sink" warning:

```suggestion
  return path.resolve(baseDir, files.at(indexOfFoundFile))
```